librz/include/rz_bin.h

```diff
@@ -846,7 +846,7 @@ RZ_API void rz_bin_symbol_free(RzBinSymbol *sym);
 static inline bool rz_bin_reloc_has_target(RzBinReloc *reloc) {
 	return reloc->target_vaddr && reloc->target_vaddr != UT64_MAX;
 }
-RZ_API void rz_bin_reloc_free(RzBinReloc *reloc);
+RZ_API void rz_bin_reloc_free(RZ_NONNULL  RzBinReloc *reloc);
```

Comment:
This is wrong because a `_free` function should accept NULL and it should just do nothing in that case, similarly to `free(NULL)`.